src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -68,6 +78,22 @@ def get_wf_by_run_id(
 
         return wf_run.get_status_model()
 
+    def get_task_run_id(
+        self,
+        wf_run_id: WorkflowRunId,
+        task_inv_id: TaskInvocationId,
+        config_name: ConfigName,
+    ) -> TaskRunId:
+        wf_run_model = self.get_wf_by_run_id(
+            wf_run_id=wf_run_id, config_name=config_name
+        )
+        # TODO: figure out what happens when there's no matching task run.
```

Comment:
Issue: hanging TODO. Either fix or create new Jira ticket and attach it here

Reply:
Fixed!